src/oc_erchef/apps/oc_chef_wm/src/oc_chef_wm_sup.erl

```diff
@@ -98,6 +98,9 @@ maybe_start_action(false, Workers) ->
 
 
 check_actions_queue_at_capacity(PoolNameAtom, Vhost, Queue) ->
+    %% TODO(ssd) 2020-08-10: This whole function is now silly. This fixes dialyzer, but
+    %% really the whole thing can be removed, but really really all of the rabbitmq
+    %% management code can be removed SOON (TM).
```

Comment:
We probably don't need this comment anymore since we no longer have other changes in this function

Reply:
Fixed.